com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/protocol/Requests.java

```diff
@@ -60,6 +60,12 @@
         externalTerminal;
     }
 
+    public static enum CliStyle {
```

Comment:
> Because they are constants, the names of an enum type's fields are in uppercase letters.

See https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html for reference

Reply:
Nice.